### PR TITLE
feat: add helm rollback action

### PR DESCRIPTION
**Pull request by bryanspears**
## Description

Devs need the ability to rollback both bad and failed deployments.

### CODE REVIEW

File: .github/workflows/helm-rollback.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          kubectl config set-credentials deploy --token="${{ secrets.k8s_token }}"
+          kubectl config set-context default --cluster=k8s --user=deploy
+          kubectl config use-context default
+          helm version
+          kubectl version
+          DEPLOYMENT="${{ github.event.repository.name }}"
+          if [ ! -z "${{ inputs.deployment }}" ]; then
+            DEPLOYMENT="${{ inputs.deployment }}"
+          fi
+          helm upgrade ${DEPLOYMENT} -n ${{ inputs.namespace }}
```
Comment (Jesse0Michael) [inline .github/workflows/helm-rollback.yaml:48]: is this supposed to be `helm rollback ...`

Comment (bryanspears) [reply .github/workflows/helm-rollback.yaml]: Sigh.... my head is full of snot at the moment. Thanks.

**Review by Jesse0Michael — APPROVED** [review]

File: .github/workflows/helm-rollback.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+name: Helm Rollback
+on:
+  workflow_call:
+    inputs:
+      namespace:
+        description: "Kubernetes namespace to deploy resources in"
```
Comment (paulredmond) [inline .github/workflows/helm-rollback.yaml:6]: Nit: Would the wording "rollback" be more fitting here:

```suggestion
        description: "Kubernetes namespace scope for this rollback"
```
/

File: .github/workflows/helm-rollback.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        required: false
+        default: api
+        type: string
+      environment:
+        description: "Target environment for deployment (stg|prod)"
```
Comment (paulredmond) [inline .github/workflows/helm-rollback.yaml:11]:
```suggestion
        description: "Target environment for rollback (stg|prod)"
```

**Review by paulredmond — APPROVED** [review]
LGTM

**Review by paulredmond — APPROVED** [review]

**Review by aspencer — APPROVED** [review]

**Pull request merged**